### PR TITLE
updated code to show 96.3% as per tutorial

### PR DESCRIPTION
**Pull request by revanthsamavedam**
The current mischief.js file shows 100% code coverage when initially run. I have updated the code to show 96.3% as per the tutorial.

### CODE REVIEW

File: mischief.js [hunk 1]
```diff
@@ -33,19 +35,18 @@ function transferMoney(fromAccount, toAccount, amount, description) {
...
-account3 = []; // rogue developer account
+let account1 = [];
+let account2 = [];
+let account3 = []; // rogue developer account
 
 console.log("\nExpect Account1 Opening Balance       "+getAccountBalance(account1) +" === 0        \u2713 ");
-creditAccount(account1, 20000, 'Add Funds');
-transferMoney(account1,account2, 100, 'Give money to friend')
-transferMoney(account2,account1, 10, 'Transfer back 10')
+creditAccount(account1, 5000, 'Add Funds');
```
Comment (nelsonic) [inline mischief.js:45]: do we _need_ to change this from `20000` to `5000` ?

**Review by nelsonic — COMMENTED** [review]
@durja PR looks good. I just had one question see above ⬆️

File: mischief.js (hunk 1 above)
Comment (revanthsamavedam) [reply mischief.js:45]: @nelsonic - It has to be an input less than 10000 to show the 96.3% for the first time we run the coverage report. Please let me know.

Comment (nelsonic) [reply mischief.js:45]: @durja you're right. well-spotted. thanks for opening this PR! 👍

**Review by nelsonic — APPROVED** [review]
@durja well-spotted and thanks for updating! 👍

**Pull request merged**